person.rb

```diff
@@ -1,23 +1,23 @@
 class Person
   attr_reader :id
   attr_accessor :name, :age
-  
-  def initialize( age, *)
-    @id = rand(0..1000000)
+
+  def initialize(age, *)
+    @id = rand(0..1_000_000)
     @age = age
-    @name ||= "Unknown"
+    @name ||= 'Unknown'
     @parent_permission ||= true
   end
 
   private
 
-  def is_of_age?
-    @age >= 18 ? true : false 
+  def of_age?
```

Comment:
- [OPTIONAL] This function's name is incorrect. It should be `is_of_age?` Because of the typo, we get an undefined method name error at line 21. Let's update the method name.

Reply:
Thank you for your review, Muhammed. I changed the name method name because of robocop lint. When I use the name as "is_of_age?"; I got the below linter error, that's why I changed the name to "of_age?"

![Screen Shot 2022-03-01 at 03 09 43](https://user-images.githubusercontent.com/24368914/156080284-5499525c-7352-4dc5-99ac-e5f2e31873eb.png)